src/irmin-pack/unix/file_manager.ml

```diff
@@ -195,6 +195,18 @@ struct
     Stats.incr_fm_field Flush;
     flush_index_and_its_deps ?hook t
 
+  (* Explicit fsync ********************************************************* *)
+
+  let fsync t =
+    let open Result_syntax in
+    let* _ = Dict.fsync t.dict in
+    let* _ = Control.fsync t.control in
+    let* _ = Suffix.fsync t.suffix in
```

Comment:
```suggestion
    let* _ = Suffix.fsync t.suffix in
    let* _ = Control.fsync t.control in
```
in the very improbable situation where the process crashes just between two fsyncs calls, I think this is the order that ensure that the files are still readable when opened.

Reply:
Ah, good catch. I was trying to follow the flush order but now see the control file is fsynced after both dict and suffix. Will make this change.